Jenkinsfile.integration

```diff
@@ -109,6 +109,7 @@ pipeline {
                 timeout(time: 10, unit: 'MINUTES', activity: true)
             }
             steps {
+                sh 'touch /tmp/$SOCOK8S_ENVNAME.needcleanup'
```

Comment:
should we use `${SOCOK8S_ENVNAME}` in case there is issues? *

* /me not a bash expert

Reply:
For safety, I would do it. It's clearer. I think this works, though.